pynamodb/attributes.py

```diff
@@ -288,6 +307,15 @@ def _dynamo_to_python_attr(cls, dynamo_key: str) -> str:
         """
         return cls._dynamo_to_python_attrs.get(dynamo_key, dynamo_key)  # type: ignore
 
+    @classmethod
+    def _get_discriminator_attribute(cls) -> Optional['DiscriminatorAttribute']:
+        return cls.get_attributes()[cls._discriminator] if cls._discriminator else None  # type: ignore
```

Comment:
```suggestion
        return cast(DiscriminatorAttribute , cls.get_attributes()[cls._discriminator]) if cls._discriminator else None
```

Reply:
I can add the cast but will still need `# type: ignore` since mypy doesn't know how to get class attributes that are created by the metaclass